test/functional/Java22andUp/src/org/openj9/test/jep454/downcall/PrimitiveTypeTests2.java

```diff
@@ -263,7 +265,7 @@ public void test_strlenFromDefaultLibWithMemAddr_2() throws Throwable {
 		MemorySegment strlenSymbol = defaultLibLookup.find("strlen").get();
 		FunctionDescriptor fd = FunctionDescriptor.of(JAVA_LONG, ADDRESS);
 		MethodHandle mh = linker.downcallHandle(fd);
-		MemorySegment funcSegmt = arena.allocateFrom("JEP454 DOWNCALL TEST SUITES");
+		MemorySegment funcSegmt = PrimitiveTypeTests1.allocateString("JEP424 DOWNCALL TEST SUITES");
```

Comment:
424 should probably be 454 here as well.

Reply:
Fixed.